haiku-apps/vcdgear/vcdgear-1.1.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool to go the way from the VideoCD (VCD) to the MPEG file"
+DESCRIPTION="Since BeOS is supposed to be the multimedia operating system, \
+it also should be able to deal with hardcopies of media files. \
+In our case these are VideoCD's, containing one or more MPEG-1 \
+video files on a CD-ROM (XA) as laid down in the white book \
+of compact disc standards. Since BeOS already has the media \
+player supplied that is capable of playing MPEG streams, we \
+needed a tool to go the way from the VideoCD (VCD) to the MPEG \
+file on our harddisk. This can be done by this tool, called \
+VCDGear, originally coded by Dracore (dracore@home.com), and \
+ported to BeOS R5 by myself, Be-NiB (be-nib@web.de)."
+HOMEPAGE="http://www.vcdgear.com"
+SOURCE_URI="http://www.vcdgear.com/files/VCDGear_1.1_beos5.zip"
+CHECKSUM_SHA256="21e91fc079e1a7db4c453dd72f92d978eb444f5fc7d09f091bb9834dc8c6c778"
+LICENSE="MIT"
+COPYRIGHT="2000 Dracore
+	Be-NiB"
+REVISION="1"
+SOURCE_DIR="VCDGear"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
+
+PROVIDES="
+	vcdgear = $portVersion
+	cmd:VCDGear
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES=""
+
+BUILD()
+{
+	true
+}
+
+INSTALL()
+{
+	mkdir -p $binDir $docDir
+	cp -f changes $docDir
+	cp -f readme $docDir
+	chmod +x VCDGear
+	cp -f VCDGear $binDir
```

Comment:
Can you swap the 2 last line? You have to adjust the path for chmod!

Reply:
swapped (thanks)